src/anomalib/engine/engine.py

```diff
@@ -882,7 +884,11 @@ def export(
             input_size (tuple[int, int] | None, optional): A statis input shape for the model, which is exported to ONNX
                 and OpenVINO format. Defaults to None.
             transform (Transform | None, optional): Input transform to include in the exported model. If not provided,
-                the engine will try to use the transform from the datamodule or dataset. Defaults to None.
+                the transform is taken from the model. Defaults to None.
+            compression_type (CompressionType | None, optional): Compression type for OpenVINO exporting only.
+                Defaults to None.
+            datamodule (AnomalibDataModule | None, optional): Lightning datamodule.
```

Comment:
To be inline with the rest of the CLI entrypoints, can you add `datamodule` to https://github.com/openvinotoolkit/anomalib/blob/395c5f631b86e7b977b15b381797f96f245f4541/src/anomalib/cli/cli.py#L235 similar to https://github.com/openvinotoolkit/anomalib/blob/395c5f631b86e7b977b15b381797f96f245f4541/src/anomalib/cli/cli.py#L217
This ignores adding `datamodule` to the CLI arguments from the method.
Can you also add 
```python
parser.add_argument(
            "--data",
            type=AnomalibDataModule,
            required=False,
        )
```
This will register datamodule under the `data` key.

Reply:
I hope this is how it should look like

Reply:
Perfect! Thanks
<img width="1230" alt="image" src="https://github.com/openvinotoolkit/anomalib/assets/17232914/dfb02a90-c71e-4e7e-a5fa-31bda1a0fa00">